Alignment/OfflineValidation/test/testPVValidation_Relvals_DATA.ini

```diff
@@ -24,11 +28,21 @@ bowtag: TrackerSurafceDeformations_v1_express
 vertextype: offlinePrimaryVertices
 tracktype: ALCARECOTkAlMinBias
 
+[Refit]
+refittertype: RefitType.COMMON
+ttrhtype: WithTrackAngle
+#ttrhtype: WithAngleAndTemplate
+
 [Selection]
 applyruncontrol: True
 ptcut: 3.
 runboundary: 275657 
-lumilist: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt 
+# also on afs
+#lumilist: /afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt
+lumilist: /eos/cms/store/group/comm_dqm/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt
```

Comment:
@smuzaffar out of curiosity, the cms-bot does not see this eos partition?
I get this error in the unit tests [logs](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b6c432/9790/unitTests.log):
```bash
Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 1029, in <module>
    main()
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 790, in main
    if not isInJSON(run,lumilist[0]):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-10-07-1100/bin/slc7_amd64_gcc820/submitPVValidationJobs.py", line 161, in isInJSON
    with open(jsonfile, 'rb') as myJSON:
IOError: [Errno 2] No such file or directory: u'/eos/cms/store/group/comm_dqm/certification/Collisions16/13TeV/DCSOnly/json_DCSONLY.txt'
Failure running PV Validaiton run-by-run submission: status 1
status = 256
```